pkg/windows/build_python.ps1

```diff
@@ -31,12 +34,12 @@ param(
     # The version of Python to be built. Pythonnet only supports up to Python
     # 3.8 for now. Pycurl stopped building wheel files after 7.43.0.5 which
     # supported up to 3.8. So we're pinned to the latest version of Python 3.8.
-    # We may have to drop support for pycurl.
+    # We may have to drop support for pycurl or build it ourselves.
     # Default is: 3.8.14
     [String] $Version = "3.8.14",
```

Comment:
3.8.15 was recently released (security release). Can we bump the version here

Reply:
Absolutely. Though it's just a placeholder for now. Relenv has the version hard-coded